conda/environment.devenv.yml

```diff
@@ -92,7 +92,6 @@ dependencies:
 - pyyaml
 - qt
```

Comment:
I think this needs to be removed too, if I'm not mistaken it pulls in qt-webengine as dependency